src/configStore.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Config redux store
+ *
+ * @author Vinh Le <lethanhvinh95@gmail.com>
+ *
+ */
+
+import { createStore, applyMiddleware, compose } from 'redux'
+import { rootReducer } from './reducer'
+const { createLogger } = require('redux-logger')
```

Comment:
This lib is weirdly implemented with TS:
https://github.com/LogRocket/redux-logger/issues/89

Reply:
change "target": "es5" to "target": "esnext" in tsconfig.json

import {createLogger} from "redux-logger"